### ISSUE TITLE
[a11y] - screen reader - missing status message for editing and editional information

### ISSUE THREAD

**Issue by transkamp** [open-pioneer/trails-openlayers-base-packages#435]
### Issue description

https://open-pioneer.github.io/trails-demo/openlayers-base-packages/dev/samples/showcase/?demo=editing

1) as a screen reader user, i would like to receive information about the status of the editing. to do this, the attribute `aria-live=“polite” `must be attached to the html  div element.

![Image](https://github.com/user-attachments/assets/d7ad42be-06f8-490b-9633-cc9b513f66a7)

#####
2) it also makes sense for screan reader users to inform the user that the editing function is not available for this target group.
#####

3) although a tooltip is displayed in the a11y tree, it is not actually read out
![Image](https://github.com/user-attachments/assets/3f60ba00-0d3a-4c8d-8cdb-45322d9bef12)

####
4) unfortunately, the button bar cannot be used on mobile devices because the button bar does not change automatically (see screenshot above)

### Validations

- [x] Read and follow our [code of conduct](https://github.com/open-pioneer/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [contributing guidelines](https://github.com/open-pioneer/trails-starter/blob/main/docs/Contributing.md)
- [x] Check that there isn't already a similar issue

**Comment by mbeckem** [comment]
Our editing functionality is pretty barebones at the moment. I'd like to defer a11y improvements in this area until we get a better editing experience merged (see https://github.com/open-pioneer/trails-openlayers-base-packages/pull/387). 

Tooltips also need major improvements in the future, I don't think that it makes sense to make a11y enhancements for the current implementations (see https://github.com/open-pioneer/trails-openlayers-base-packages/issues/303).
